templates/build/news-feed-filters.build.hbs

```diff
@@ -14,8 +14,27 @@
         <div id="collapse-\{{id}}-\{{@index}}" class="panel-collapse collapse">
           <div class="panel-body">
             <fieldset data-filter-group>
-              \{{#each [data]}}
-              <div class="btn hidden-filter-controls-filter focus-outline" tabindex="0" data-toggle="\{{[class]}}" data-field="\{{../name}}" data-value="\{{[name]}}" >\{{[name]}}</div>
+              \{{#each  ../detailViewOptions}}
+                \{{#ifCond type '==' 'date'}}
+                  \{{#each ../data}}
+                    \{{#ifCond ../../name '==' 'Date'}}
+                      \{{#if @first}}
+                        <div class="date-picker-container">
+                          <i class="fa fa-calendar">&nbsp;</i>
+                          <input type="text" id="dateFrom" value="" class="date-picker btn overlay-filter hidden-filter-controls-filter-date-range focus-outline" data-toggle="dateFrom" data-field="\{{../../name}}" data-value="">
+                          to
+                        </div>
+                        <div class="date-picker-container">
+                          <i class="fa fa-calendar">&nbsp;</i>
+                          <input type="text" id="dateTo" value="" class="date-picker btn overlay-filter hidden-filter-controls-filter-date-range focus-outline" data-toggle="dateTo" data-field="\{{../../name}}" data-value="">
+                          <input type="button" value="Reset" id="resetDate" class="btn btn-link btn-link-reset hidden">
```

Comment:
2 errors here.

1. You must not use unique `id`s in the HTML markup unless it is specific to the widget instance. That's because if we have multiple instances (or in this case, multiple date filters), then there will be multiple elements with `id="resetDate"` and that's invalid HTML. You can't have more that one element on the page with the same `id` as it would produce unexpected results.
2. Use kebab-case for HTML classes and IDs.